.yo-rc.json

```diff
@@ -32,9 +32,7 @@
     },
     "app": "ordino",
     "libraryAliases": {
-      "d3": "d3/d3",
-      "scrollTo": "jquery.scrollto/jquery.scrollTo.js",
-      "jquery": "jquery/dist/jquery"
+      "d3": "d3/d3"
```

Comment:
This is not used anymore in the new webpack config, so it can be safely removed :+1: